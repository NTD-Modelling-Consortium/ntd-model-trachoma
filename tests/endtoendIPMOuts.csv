Time,age_start,age_end,measure,draw_0
2019,None,None,nSurvey,0.0
2019,None,None,surveyPass,0.0
2019,None,None,trueElimination,0.0
2019,0,100,nDosesMDA,0.0
2019,0,100,MDAcoverage,0.0
2019,0,100,numMDAs,0.0
2019,1,10,nDosesMDA,0.0
2019,1,10,MDAcoverage,0.0
2019,1,10,numMDAs,0.0
2020,None,None,nSurvey,0.0
2020,None,None,surveyPass,0.0
2020,None,None,trueElimination,0.0
2020,0,100,nDosesMDA,807.0
2020,0,100,MDAcoverage,0.8094282848545636
2020,0,100,numMDAs,1.0
2020,1,10,nDosesMDA,0.0
2020,1,10,MDAcoverage,0.0
2020,1,10,numMDAs,0.0
2021,None,None,nSurvey,0.0
2021,None,None,surveyPass,0.0
2021,None,None,trueElimination,0.0
2021,0,100,nDosesMDA,810.0
2021,0,100,MDAcoverage,0.811623246492986
2021,0,100,numMDAs,1.0
2021,1,10,nDosesMDA,0.0
2021,1,10,MDAcoverage,0.0
2021,1,10,numMDAs,0.0
2022,None,None,nSurvey,1.0
2022,None,None,surveyPass,0.0
2022,None,None,trueElimination,0.0
2022,0,100,nDosesMDA,794.0
2022,0,100,MDAcoverage,0.7955911823647295
2022,0,100,numMDAs,1.0
2022,1,10,nDosesMDA,0.0
2022,1,10,MDAcoverage,0.0
2022,1,10,numMDAs,0.0
2023,None,None,nSurvey,0.0
2023,None,None,surveyPass,0.0
2023,None,None,trueElimination,0.0
2023,0,100,nDosesMDA,796.0
2023,0,100,MDAcoverage,0.796
2023,0,100,numMDAs,1.0
2023,1,10,nDosesMDA,0.0
2023,1,10,MDAcoverage,0.0
2023,1,10,numMDAs,0.0
2024,None,None,nSurvey,0.0
2024,None,None,surveyPass,0.0
2024,None,None,trueElimination,0.0
2024,0,100,nDosesMDA,850.0
2024,0,100,MDAcoverage,0.8508508508508509
2024,0,100,numMDAs,1.0
2024,1,10,nDosesMDA,455.0
2024,1,10,MDAcoverage,1.6635918825699847
2024,1,10,numMDAs,2.0
2025,None,None,nSurvey,0.0
2025,None,None,surveyPass,0.0
2025,None,None,trueElimination,0.0
2025,0,100,nDosesMDA,875.0
2025,0,100,MDAcoverage,0.875
2025,0,100,numMDAs,1.0
2025,1,10,nDosesMDA,476.0
2025,1,10,MDAcoverage,1.7404347477340179
2025,1,10,numMDAs,2.0
2026,None,None,nSurvey,0.0
2026,None,None,surveyPass,0.0
2026,None,None,trueElimination,0.0
2026,0,100,nDosesMDA,874.0
2026,0,100,MDAcoverage,0.874
2026,0,100,numMDAs,1.0
2026,1,10,nDosesMDA,484.0
2026,1,10,MDAcoverage,1.6863473562799733
2026,1,10,numMDAs,2.0
2027,None,None,nSurvey,0.0
2027,None,None,surveyPass,0.0
2027,None,None,trueElimination,0.0
2027,0,100,nDosesMDA,862.0
2027,0,100,MDAcoverage,0.862
2027,0,100,numMDAs,1.0
2027,1,10,nDosesMDA,480.0
2027,1,10,MDAcoverage,1.6902361911846755
2027,1,10,numMDAs,2.0
2028,None,None,nSurvey,1.0
2028,None,None,surveyPass,0.0
2028,None,None,trueElimination,0.0
2028,0,100,nDosesMDA,863.0
2028,0,100,MDAcoverage,0.8638638638638638
2028,0,100,numMDAs,1.0
2028,1,10,nDosesMDA,509.0
2028,1,10,MDAcoverage,1.764471945212063
2028,1,10,numMDAs,2.0
