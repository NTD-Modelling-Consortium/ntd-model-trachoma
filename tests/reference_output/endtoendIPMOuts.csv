Time,age_start,age_end,measure,draw_0
2019,None,None,nSurvey,0.0
2019,None,None,surveyPass,1.0
2019,None,None,trueElimination,0.0
2019,0,100,nDosesMDA,0.0
2019,0,100,MDAcoverage,0.0
2019,0,100,numMDAs,0.0
2019,1,10,nDosesMDA,0.0
2019,1,10,MDAcoverage,0.0
2019,1,10,numMDAs,0.0
2020,None,None,nSurvey,0.0
2020,None,None,surveyPass,1.0
2020,None,None,trueElimination,0.0
2020,0,100,nDosesMDA,0.0
2020,0,100,MDAcoverage,0.0
2020,0,100,numMDAs,0.0
2020,1,10,nDosesMDA,0.0
2020,1,10,MDAcoverage,0.0
2020,1,10,numMDAs,0.0
2021,None,None,nSurvey,0.0
2021,None,None,surveyPass,1.0
2021,None,None,trueElimination,0.0
2021,0,100,nDosesMDA,0.0
2021,0,100,MDAcoverage,0.0
2021,0,100,numMDAs,0.0
2021,1,10,nDosesMDA,0.0
2021,1,10,MDAcoverage,0.0
2021,1,10,numMDAs,0.0
2022,None,None,nSurvey,1.0
2022,None,None,surveyPass,0.0
2022,None,None,trueElimination,0.0
2022,0,100,nDosesMDA,799.0
2022,0,100,MDAcoverage,0.7997997997997998
2022,0,100,numMDAs,1.0
2022,1,10,nDosesMDA,0.0
2022,1,10,MDAcoverage,0.0
2022,1,10,numMDAs,0.0
2023,None,None,nSurvey,0.0
2023,None,None,surveyPass,0.0
2023,None,None,trueElimination,0.0
2023,0,100,nDosesMDA,813.0
2023,0,100,MDAcoverage,0.8138138138138138
2023,0,100,numMDAs,1.0
2023,1,10,nDosesMDA,0.0
2023,1,10,MDAcoverage,0.0
2023,1,10,numMDAs,0.0
2024,None,None,nSurvey,0.0
2024,None,None,surveyPass,0.0
2024,None,None,trueElimination,1.0
2024,0,100,nDosesMDA,868.0
2024,0,100,MDAcoverage,0.8688688688688688
2024,0,100,numMDAs,1.0
2024,1,10,nDosesMDA,468.0
2024,1,10,MDAcoverage,1.7204763256423772
2024,1,10,numMDAs,2.0
2025,None,None,nSurvey,0.0
2025,None,None,surveyPass,0.0
2025,None,None,trueElimination,1.0
2025,0,100,nDosesMDA,828.0
2025,0,100,MDAcoverage,0.828
2025,0,100,numMDAs,1.0
2025,1,10,nDosesMDA,488.0
2025,1,10,MDAcoverage,1.7521982414068744
2025,1,10,numMDAs,2.0
2026,None,None,nSurvey,0.0
2026,None,None,surveyPass,0.0
2026,None,None,trueElimination,1.0
2026,0,100,nDosesMDA,857.0
2026,0,100,MDAcoverage,0.857
2026,0,100,numMDAs,1.0
2026,1,10,nDosesMDA,490.0
2026,1,10,MDAcoverage,1.7406299689558566
2026,1,10,numMDAs,2.0
2027,None,None,nSurvey,0.0
2027,None,None,surveyPass,0.0
2027,None,None,trueElimination,1.0
2027,0,100,nDosesMDA,853.0
2027,0,100,MDAcoverage,0.8547094188376754
2027,0,100,numMDAs,1.0
2027,1,10,nDosesMDA,503.0
2027,1,10,MDAcoverage,1.737471363115693
2027,1,10,numMDAs,2.0
2028,None,None,nSurvey,1.0
2028,None,None,surveyPass,1.0
2028,None,None,trueElimination,1.0
2028,0,100,nDosesMDA,0.0
2028,0,100,MDAcoverage,0.0
2028,0,100,numMDAs,0.0
2028,1,10,nDosesMDA,0.0
2028,1,10,MDAcoverage,0.0
2028,1,10,numMDAs,0.0
